actioncable/lib/action_cable/connection/base.rb

```diff
@@ -96,6 +99,11 @@ def dispatch_websocket_message(websocket_message) # :nodoc:
       end
 
       def handle_channel_command(payload)
+        if payload["type"] == ActionCable::INTERNAL[:message_types][:pong]
+          process_pong(payload)
+          return
+        end
```

Comment:
Since it's not a command, I would move it out of the `#handle_channel_command` method into the `#dispatch_websocket_message`. Or, maybe, even right into the `#receive` method — there is not reason to process pongs asynchronously, we can update timestamps right away, this operation is cheap.